lib/atomic_web/views/helpers.ex

```diff
@@ -159,6 +159,34 @@ defmodule AtomicWeb.ViewUtils do
     |> URI.to_string()
   end
 
+  @doc ~S"""
```

Comment:
```suggestion
  @doc """
```
> Elixir supports some sigil variants to deal with escaping characters and interpolation. In particular, uppercase letters sigils do not perform interpolation nor escaping. For example, although both ~s and ~S will return strings, the former allows escape codes and interpolation while the latter does not.

We don't need this there, then.